.pre-commit-config.yaml

```diff
@@ -585,22 +585,22 @@ repos:
       - id: pylint
         name: Run pylint for main code
         language: system
-        entry: "./scripts/ci/pre_commit/pre_commit_pylint.sh --rcfile pylintrc"
+        entry: "./scripts/ci/pre_commit/pre_commit_pylint.sh"
```

Comment:
We don't need to specify this, `pylintrc` is one of the defaults places it looks https://github.com/PyCQA/pylint/blame/41d70221cdce96a209a479e349bdb70180bb5e8e/pylint/config/find_default_config_files.py#L27-L63